Makefile

```diff
@@ -130,6 +131,15 @@ vet:
 		exit 1; \
 	fi
 
+.PHONY: lint
+lint:
+ifeq (, $(shell which golangci-lint))
+	$(info golangci-lint can't be found, please install it: https://golangci-lint.run/usage/install/)
```

Comment:
should this just try to install it?

Reply:
@srebhan made a good point that we might not want to install anything without prompting the user. I also want to have this download the release binary instead of installing from source like it is doing now (should be faster).

Reply:
@ssoroka I vote for not installing it automatically as it's the machine of the user. There is a link to the upstream install instructions that should suffice for anyone building/developing for telegraf I guess. :-)